### ISSUE TITLE
Bomb Squad needs a proper CLI implementation

### ISSUE THREAD

**Issue by cboggs** [Fresh-Tracks/bomb-squad#16]
The existing Bomb Squad CLI was written under the strictest practices of ICDD (Imminent Conference Driven Development), hence its rather crude state.

It should use something like Kingpin or another appropriate library to handle its CLI lifecycle more sanely.

**Comment by pcarranza** [comment]
meh, only if really need. Is it needed or would it mean having another dependency? what level of complexity is needed from the CLI?

**Comment by cboggs** [comment]
True, I forget that it doesn't really need much more than a flag and an argument. Just something as simple as `-list` and `-unsilence=metric.labelname` would suffice, and the built-in `flags` package works fine for that.
